docs/09.EXT-REFERENCE-ARG.md

```diff
@@ -206,6 +206,36 @@ jerryx_arg_transform_args (const jerry_value_t *js_arg_p,
 - [jerryx_arg_transform_this_and_args](#jerryx_arg_transform_this_and_args)
 
 
+## jerryx_arg_transform_object_properties
+
+**Summary**
+
+Validate the properties of a target JS object and assign them to the native arguments.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerryx_arg_transform_object_properties (const jerry_value_t taregt,
```

Comment:
second thought: is 'target' the best name for this parameter? it is too generic, a bit non-descriptive, e.g., it does not reveal that the function expects an object here.